assets/javascript/progressive-reveal.js

```diff
@@ -30,35 +33,83 @@ function inputClicked(e, target) {
                     helpers.addClass(toggle, hiddenClass);
                 }
             }
+        });
+    };
+}
+
+function selectChangeListener(select) {
+    return function () {
+        var currentOption = select.options[select.selectedIndex],
+            currentSectionToggleId = currentOption.getAttribute(toggleAttr),
+            currentSectionToggle = document.getElementById(currentSectionToggleId);
+
+        _.each(select.options, function (option) {
+            option.setAttribute('aria-expanded', 'false');
```

Comment:
Would setting `aria-expanded` only on `options`s with `data-toggle` be better here?